App/Sagas/TextileSagas.js

```diff
@@ -265,8 +255,8 @@ export function * photosTask () {
     // Convert all our new entries to thumbs before anything else
     for (let photo of photos.reverse()) {
       try {
-        photo = yield call(scalePhoto, photo)
-        const multipartData = yield call(IPFS.addImageAtPath, photo.path, photo.thumbPath, 'default')
+        photo = yield call(getPhotoPath, photo)
+        const multipartData = yield call(IPFS.addPhoto, photo.path, 'default', '')
```

Comment:
Always suspicious about using empty strings... why not `null`?

Reply:
agreed yeah... in this case though, it's just the caption. an "add" from the mobile perspective is really 1) add the file to IPFS, 2) essentially "share" it to a thread (in this case default). I left the caption arg here because at some point you may not want to _first_ add a photo to your default thread, maybe you want to just send it direct to some other thread. at some point the go layer would have to convert `nil` to "", encrypt that and add it to the block as a file

Reply:
yep, fair enough, can change it np